frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -415,11 +414,7 @@ export class HtmlDocument extends React.Component {
     try {
       newDocumentForApiWithMention = wrapMentionsInSpanTags(state.content.raw_content)
     } catch (e) {
-      if (e === EXCEPTION_MENTION_PARSING) {
-        this.sendGlobalFlashMessage(props.t('Error while detecting the mentions'))
-      } else {
-        this.sendGlobalFlashMessage(props.t('Error while saving the new version'))
-      }
+      this.sendGlobalFlashMessage(e.message ? e.message : props.t('Error while saving the new version'))
```

Comment:
Suggestion, but it is a matter of taste.

```suggestion
      this.sendGlobalFlashMessage(e.message || props.t('Error while saving the new version'))
```

Note for myself: `e.message` is already localized.